docs/content/latest/explore/cluster-management/point-in-time-recovery-ysql.md

```diff
@@ -6,10 +6,11 @@ description: Restore data from a specific point in time in YugabyteDB for YSQL
 beta: /latest/faq/general/#what-is-the-definition-of-the-beta-feature-tag
 aliases:
 - /latest/explore/backup-restore/point-in-time-restore-ysql
+- /latest/explore/backup-restore/point-in-time-restore
 menu:
   latest:
-    identifier: explore-point-in-time-recovery
-    parent: explore
+    identifier: cluster-management-point-in-time-recovery-ysql
+    parent: explore-cluster-management
     weight: 704
 isTocNested: true
 showAsideToc: true
```

Comment:
The two tab links (YSQL/YCQL) need to be updated

Reply:
Fixed